src/components/Explore/Explore.tsx

```diff
@@ -227,15 +237,20 @@ const ExploreCopy: React.FunctionComponent<{
     const hasData = some(chartSeries, ({ data }) => data.length > 0);
     const hasMultipleLocations = selectedLocations.length > 1;
 
-    const [period, setPeriod] = useState<Period>(Period.ALL);
-    const allPeriodLabels = getAllPeriodLabels();
-    const dateRange = getDateRange(period);
+    const { pathname } = useLocation();
 
-    const onSelectPeriod = (newPeriod: Period) => {
-      const newPeriodLabel = periodMap[newPeriod].label;
-      setPeriod(newPeriod);
-      trackExploreEvent(EventAction.SELECT, `Period: ${newPeriodLabel}`);
-    };
+    // Cliking a sparkline sets the default metric, so when we navigate to a different
+    // location page, we need to force reset the metric to cases in order to override that
+    // default metric setting
+    // (We also reset the time range to 'all time')
+    useEffect(() => {
+      if (pathname.includes('/explore')) {
+        const timeoutId = scrollToExplore();
+        return () => clearTimeout(timeoutId);
+      }
+      setCurrentMetric(ExploreMetric.CASES);
+      setPeriod(Period.ALL);
+    }, [pathname, scrollToExplore]);
```

Comment:
just an edit to an existing useEffect (prev on line 200 in the orig part of the diff) -- adds:
```
setCurrentMetric(ExploreMetric.CASES);
setPeriod(Period.ALL);
```

(explanation in the added comment^^)